Application/Supporting Files/Base.lproj/Localizable.strings

```diff
@@ -30,3 +30,9 @@
 
 /* The title of the tab bar item for the settings navigation flow. */
 "settings_navigation.tab_bar_item.title" = "Settings";
+
+/* Food enum names. */
```

Comment:
Let's add a comment above each string and new line separate them. When using an XLIFF generation, `food.beans` would have this comment and the rest would have none.